public/index.html

```diff
@@ -62,7 +62,10 @@
     />
     <!-- OG tags require absolute url for images -->
     <meta name="twitter:image" content="https://excalidraw.com/og-image.png" />
-
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="block-all-mixed-content; child-src 'none'; connect-src http: ws:; default-src 'self'; font-src 'self' data: https: filesystem:; img-src 'self' data: https:; script-src 'self' 'unsafe-inline' https://www.googletagmanager.com https://www.google-analytics.com; style-src 'self' 'unsafe-inline' https:;"
```

Comment:
I'm no CSP expert.. but isn't having just `http:` and `ws:` is very loose?

Reply:
I Agree, it should be https: and wss:, as we shouldn't ideally allow fetch, XHR and websocket connections over unencrypted connections.